src/Resources/config/services/decorators.xml

```diff
@@ -90,21 +90,21 @@
         </service>
 
         <service
```

Comment:
What do you mean with missing? Shopware comes with [Shopware\\Elasticsearch\\](https://github.com/shopware/platform/blob/trunk/composer.json#L370) by default. It would need to get removed manually, right?

Reply:
I've renamed the ElasticSearch directory and executed `composer remove shopware/elasticsearch` to match your scenario. Findologic search works, no error regarding undefined service etc.
![Screen Shot 2022-03-15 at 10 45 52](https://user-images.githubusercontent.com/511559/158350964-568f6bf3-fe5e-4674-9d9e-24aea6ec63f3.png)
.